### PR TITLE
phpunit update

### PR DESCRIPTION
**Pull request by kokspflanze**
added phpunit 6 support
travis update

### CODE REVIEW

**Comment by kokspflanze** [comment]
the error is from https://github.com/php-cache/integration-tests/blob/master/src/CachePoolTest.php#L17

**Comment by kokspflanze** [comment]
that error is fixed with https://github.com/zendframework/zend-eventmanager/pull/47/files

File: test/_autoload.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+use PHPUnit\Framework\TestCase;
+
+if (! class_exists('PHPUnit_Framework_TestCase')) {
+    class_alias(TestCase::class, 'PHPUnit_Framework_TestCase');
```
Comment (webimpress) [inline test/_autoload.php]: We don't need this file and this alias here. In PHPUnit 5.7 and 6.0 we already have `TestCase` class.

Comment (kokspflanze) [inline test/_autoload.php]: the problem is https://github.com/php-cache/integration-tests/blob/master/src/CachePoolTest.php#L17 that is used for the unit-tests, that test is in a external repo and use the old class.

**Comment by marc-mabe** [comment]
Thanks for working on this!
But I'm in the middle of refactoring/splitting the component and will get tons of conflicts afterwards :(
Not sure how to handle the situation best as my time is very limited.

**Comment by marc-mabe** [comment]
See #93

**Comment by kokspflanze** [comment]
@marc-mabe no problem i just want to help with the test problem, if you already have a solution we can close it.

**Comment by marc-mabe** [comment]
I guess we can merge it to master but not to develop as the refactoring will be a new mayor

**Comment by kokspflanze** [comment]
i also need a new mayor, because the phpunit part will just work on php5.6+ and i used base develop, to get the latest changes.

**Comment by marc-mabe** [comment]
Ah I see - didn't noted it on phone.
-> I'll leave your PR open and make your changes manually on refactoring

**Comment by weierophinney** [comment]
Thanks, @kokspflanze!

**Comment by weierophinney** [comment]
Rebased and merged to develop with 1bf8a61.

**Pull request closed without merging**